plugin/src/main/kotlin/com/vanniktech/maven/publish/SonatypeHost.kt

```diff
@@ -8,25 +8,36 @@ import java.io.Serializable
  *
  * https://central.sonatype.org/articles/2021/Feb/23/new-users-on-s01osssonatypeorg/
  */
-data class SonatypeHost(
+data class SonatypeHost internal constructor(
   internal val rootUrl: String,
+  internal val centralPortal: Boolean,
 ) : Serializable {
+  constructor(rootUrl: String) : this(rootUrl, centralPortal = false)
```

Comment:
can be internal, right?

Reply:
No, it's public to support custom Sonatype hosts like https://google.oss.sonatype.org/